### PR TITLE
Review of the docker recipe

### PR DESCRIPTION
**Pull request by pattacini**
We don't actually need at this level the Gitpod stuff..

I've fully reworked the file taking https://github.com/icub-training/icub-training.github.io/blob/master/dockerfiles/Dockerfile as baseline.

### CODE REVIEW

File: dockerfiles/Dockerfile [hunk 1]
```diff
@@ -69,44 +65,31 @@ RUN git clone https://github.com/robotology/robotology-superbuild.git --depth 1
...
+    make
 
 # Clean up git configuration
 RUN git config --global --unset-all user.name && \
     git config --global --unset-all user.email
 
+# Build package containing forward kinematics inspection
+RUN git clone https://github.com/icub-tech-iit/icub-forward-kinematics-inspection.git && \
+    cd icub-forward-kinematics-inspection && mkdir build && cd build && \
+    source ${ROBOTOLOGY_SUPERBUILD_INSTALL_DIR}/share/robotology-superbuild/setup.sh && \
```
Comment (pattacini) [inline dockerfiles/Dockerfile]: `source ${ROBOTOLOGY_SUPERBUILD_INSTALL_DIR}/share/robotology-superbuild/setup.sh` should let you avoid specifying `iDynTree_DIR`.

Comment (mfussi66) [reply dockerfiles/Dockerfile]: the build process fails if it is written that way. It is necessary to run the source command as 
```
/bin/bash -c "source .../.../setup.bash"
``` 

I corrected it and continued the build, but vnc is still not available on the browser.

Comment (pattacini) [reply dockerfiles/Dockerfile]: Good catch 👍🏻 
I might have some time tomorrow morning to try it out and see myself.

Comment (pattacini) [reply dockerfiles/Dockerfile]:
```suggestion
    sh -c 'source ${ROBOTOLOGY_SUPERBUILD_INSTALL_DIR}/share/robotology-superbuild/setup.sh' && \
```

**Pull request merged**